src/Mod/Part/Gui/TaskDimension.cpp

```diff
@@ -1288,8 +1288,13 @@ void PartGui::DimensionAngular::setupDimension()
   textSep->addChild(textTransform);
 
   SoFont *fontNode = new SoFont();
-  fontNode->name.setValue("defaultFont");
-  fontNode->size.setValue(30);
+  ParameterGrp::handle group = App::GetApplication().GetUserParameter().GetGroup("BaseApp/Preferences/View");
```

Comment:
This is the same code block as above in line 503. Please sort it out to a separate function or a lambda to avoid code duplication